admin/src/components/ProductList/SettingLink.js

```diff
@@ -1,37 +1,98 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Grid, GridItem } from '@strapi/design-system/Grid';
 import { Box } from '@strapi/design-system/Box';
 import { Link } from '@strapi/design-system/Link';
 import { Typography } from '@strapi/design-system/Typography';
 import ArrowRight from '@strapi/icons/ArrowRight';
+import pluginPkg from '../../../../package.json';
 
 const SettingLink = () => {
+  const [isNewVersionAvailable, setIsNewVersionAvailable] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      // call github api to get the latest version of the plugin
+      const responseGithub = await fetch(
+        'https://api.github.com/repos/manishkatyan/strapi-stripe/releases/latest'
+      );
+      const data = await responseGithub.json();
+
+      // compare the latest version with the current version
+      if (data.tag_name !== pluginPkg.version) {
+        setIsNewVersionAvailable(true);
+      }
+    })();
+  }, []);
+
   return (
     <Box>
-      <Box
-        shadow="tableShadow"
-        background="neutral0"
-        paddingTop={6}
-        paddingLeft={7}
-        paddingRight={7}
-        paddingBottom={6}
-        hasRadius
-      >
-        <Box>
-          <Grid gap={4}>
-            <GridItem col={6} s={12}>
-              <Link to="/settings/strapi-stripe" endIcon={<ArrowRight />}>
-                Manage Your Stripe Configuration
-              </Link>
-            </GridItem>
-            <GridItem col={6} s={12}>
-              <Typography variant="pi">
-                Need help? Contact us at : support@higheredlab.com
```

Comment:
support@higheredlab.com is hard coded here, Better to create contants/env.js and put it there. In future  it will be easy to change.

Reply:
sure